src/Phinx/Db/Adapter/SqlServerAdapter.php

```diff
@@ -1186,7 +1186,11 @@ protected function getColumnSqlDefinition(Column $column, $create = true)
         }
 
         if ($column->isIdentity()) {
-            $buffer[] = 'IDENTITY(1, 1)';
+        //$buffer[] = 'IDENTITY(1, 1)';
+            $seed = $column->getSeed() ? $column->getSeed() : 1;
+            $increment = $column->getIncrement() ? $column->getIncrement() : 1;
+         //$buffer[] = 'IDENTITY(' . $column->getSeed() . ',' . $column->getIncrement() . ')';
```

Comment:
- Remove comments
- `x ? x : y` can be simplified to `x ?: y`

Reply:
Done